### DISCUSSION TITLE
On exiting the full screen, the entire page re-renders

### DISCUSSION THREAD

**Discussion by suganya27302** [openseadragon/openseadragon#2486] [Q&A]
Why does exiting the full screen of one OSD affect the entire page?  The entire page is re-rendering. What is the reason for the re-rendering?

**Comment by iangilman** [comment]
I'm not sure why your page is re-rendering... That doesn't always happen. Try the fullscreen on https://openseadragon.github.io/. So it's a combination of OSD + whatever you have on your page that's causing the re-rendering.

That said, the OSD part of the equation has to do with the history of how OSD's fullscreen came to be: https://github.com/openseadragon/openseadragon/issues/1050#issuecomment-254568536. We want to move away from that (https://github.com/openseadragon/openseadragon/issues/2205) but no one has stepped up to tackle it yet.
